tests/suites/test_suite_ssl.function

```diff
@@ -3537,9 +3537,7 @@ exit:
 /* BEGIN_CASE */
 void conf_group()
 {
-    uint16_t iana_tls_group_list[] = { MBEDTLS_SSL_IANA_TLS_GROUP_SECP192R1,
-                                       MBEDTLS_SSL_IANA_TLS_GROUP_SECP224R1,
-                                       MBEDTLS_SSL_IANA_TLS_GROUP_SECP256R1,
+    uint16_t iana_tls_group_list[] = { MBEDTLS_SSL_IANA_TLS_GROUP_SECP256R1,
```

Comment:
I think here the intent of the test was to have multiple groups, and not exactly the default list. Please add one or two other groups, like `MBEDTLS_SSL_IANA_TLS_GROUP_SECP384R1`.